libs/service-portal/documents/src/utils/getOrganizationLogoUrl.ts

```diff
@@ -0,0 +1,32 @@
+interface Organization {
+  id: string
+  title: string
+  logo: null | {
+    id: string
+    url: string
+    title: string
+    width: number
+    height: number
+  }
+}
+
+const getOrganizationLogoUrl = (
+  forName: string,
+  orgs: Array<Organization>,
+): string => {
+  if (orgs.length > 0) {
+    const qs = String(forName).trim().toLocaleLowerCase()
+    const match =
+      orgs.find((o: Organization) => o.title?.toLocaleLowerCase() === qs) ||
+      orgs.find((o: Organization) => o.logo?.title === 'Skjaldarmerki')
+    const c = match?.logo?.url
+    const url =
+      c ??
+      '//images.ctfassets.net/8k0h54kbe6bj/6XhCz5Ss17OVLxpXNVDxAO/d3d6716bdb9ecdc5041e6baf68b92ba6/coat_of_arms.svg'
```

Comment:
Need to revisit, this shouldn't be hard coded here, but we're making the implementation match the apps for v1:
https://github.com/island-is/island.is/blob/a8af0e669c49008df542d2ed4cf00da829fcf0e4/apps/native/app/src/stores/organizations-store.ts#L67